wgpu-types/src/lib.rs

```diff
@@ -6462,8 +6462,14 @@ impl<L, V> TextureDescriptor<L, V> {
     /// <https://gpuweb.github.io/gpuweb/#abstract-opdef-compute-render-extent>
     #[must_use]
     pub fn compute_render_extent(&self, mip_level: u32, plane: Option<u32>) -> Extent3d {
-        let width = self.size.width >> mip_level;
-        let height = self.size.height >> mip_level;
+        let min_size = if let TextureFormat::NV12 | TextureFormat::P010 = self.format {
+            2
+        } else {
+            1
+        };
+
+        let width = u32::max(min_size, self.size.width >> mip_level);
+        let height = u32::max(min_size, self.size.height >> mip_level);
```

Comment:
thought: Fun fact, you can use this as a method:

```suggestion
        let width = (self.size.width >> mip_level).max(min_size);
        let height = (self.size.height >> mip_level).max(min_size);
```

I don't know that this makes things more _readable_, but I thought I'd throw that out there.

Reply:
imo I'd vote against this, and keep it as a mathod call.

Reply:
@cwfitzgerald: I'm confused what you're voting against. The suggestion (which _is_ a method call), or calling the method like an associated function?